_security-analytics/threat-intelligence/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Threat intelligence
+nav_order: 40
+has_children: true
+---
+
+# Threat intelligence
+
+Threat intelligence in Security Analytics offers the capability to integrate your threat intelligence feeds. Feeds are comprised Indicators of Compromise (IOCs), which search for malicious indicators within your data by setting up a threat intelligence monitor. These monitors generate findings and can send notifications when malicious IPs, domains, or hashes referenced from the threat intelligence feeds match your data.
+
+To access threat intelligence, log into OpenSearch Dashboards, and select **Security Analytics** > **Threat Intelligence**.
+
+You can interact with threat intelligence in the following ways:
+
+- Threat intelligence API]. To configure threat intelligence using API operations, see [Threat Intelligence APIs]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/api/threat-intel-api/).
+- OpenSearch Dashboards: To configure and use threat intelligence through the OpenSearch Dashboards interface, see [Getting Started]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/getting-started/).
```

Comment:
```suggestion
- OpenSearch Dashboards. To configure and use threat intelligence through the OpenSearch Dashboards interface, see [Getting Started]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/getting-started/).
```

Reply:
I would swap the two bullets above to match the order in the related articles section.

Reply:
I'll use the colon on the other one as well.